.github/workflows/main.yml

```diff
@@ -41,4 +41,4 @@ jobs:
           # Token to connect to GitHub. Use secrets.GITHUB_TOKEN
           githubToken: secrets.GITHUB_TOKEN
           # Path of the package relatively to the root of the repository
-          relativePath: # optional, default is 
+          # relativePath: # optional, default is 
```

Comment:
Is this not needed anymore?

Reply:
there wasn't any value, so it doesn't make a sence to have here unused keys right?